src/main/java/org/spongepowered/common/mixin/core/server/MixinNetHandlerPlayServer.java

```diff
@@ -187,4 +193,32 @@ public void processCommandBlock(C17PacketCustomPayload packetIn, CallbackInfo ci
 
     }
 
+    @Inject(method = "processPlayer", at = @At(value = "FIELD", target = "net/minecraft/network/NetHandlerPlayServer.hasMoved", ordinal = 2), cancellable = true, remap = false)
```

Comment:
I believe your reference to the field should be dot separated not slash separated and it needs to include the type information. eg.

`net.minecraft.network.NetHandlerPlayServer.hasMoved:Z`

Your method reference also needs to include more information see [this](https://github.com/SpongePowered/SpongeCommon/blob/206575ac29e767c6645353101303e320fe360ae6/src/main/java/org/spongepowered/common/mixin/core/entity/vehicle/MixinEntityBoat.java#L76) for an example.